cmake/OpenEXRSetup.cmake

```diff
@@ -224,6 +224,16 @@ if(OPENEXR_FORCE_INTERNAL_ZLIB OR NOT TARGET ZLIB::ZLIB)
     set(zlibpostfix "d")
   endif()
 
+  add_library(zlib_static STATIC IMPORTED GLOBAL)
```

Comment:
is this chunk a whitespace change? I'm not spotting the diff.

Reply:
Not sure I follow, lines 227-236 are new, right?

Reply:
I think the diff viewer messed up. I added the comment on a chunk that I don't see in the diff now.